src/main/java/org/embulk/input/gcs/GcsFileInputPlugin.java

```diff
@@ -62,6 +65,9 @@ else if (AuthUtils.AuthMethod.private_key.equals(task.getAuthMethod())) {
         // list files recursively if path_prefix is specified
         if (task.getPathPrefix().isPresent()) {
             task.setFiles(GcsFileInput.listFiles(task));
+            if (task.getFiles().getTaskCount() == 0) {
+                logger.info("No file is found. Confirm path_prefix option is correct");
```

Comment:
IIUC, this can happen even if `path_prefix` is "correct". Just "No file is found there" should be sufficient as a log message.

```suggestion
                logger.info("No file is found in the path(s) identified by path_prefix");
```